src/Sylius/Bundle/ApiBundle/Command/Payment/AddPaymentRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command\Payment;
+
+use Sylius\Bundle\ApiBundle\Attribute\OrderTokenValueAware;
+use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;
+
+/** @experimental */
+#[OrderTokenValueAware]
+class AddPaymentRequest implements IriToIdentifierConversionAwareInterface
```

Comment:
```suggestion
use Sylius\Bundle\ApiBundle\Attribute\OrderTokenValueAware;
use Sylius\Bundle\ApiBundle\Attribute\PaymentIdAware;
use Sylius\Bundle\ApiBundle\Command\IriToIdentifierConversionAwareInterface;

/** @experimental */
#[OrderTokenValueAware]
#[PaymentIdAware]
class AddPaymentRequest implements IriToIdentifierConversionAwareInterface
```

To complete the suggestion of @NoResponseMate just bellow.

Reply:
Adding this makes no sense as it would not be used at all, as we do not have payment id set as a parameter in the URL 🖖🏻